src/components/InteractiveStepSubHeader.tsx

```diff
@@ -49,6 +49,16 @@ function InteractiveStepSubHeader({stepNames, startStepIndex = 0, onStepSelected
         [],
     );
 
+    useEffect(() => {
```

Comment:
I added this so that when I pass `startStepIndex={screenIndex}` to `InteractiveStepSubHeader`, it rerenders. 

We want that on click on Next the completed steps are changed to done.

https://github.com/Expensify/App/assets/3069065/83ac1af7-2ae2-4735-8e62-0b4a359bb839

Reply:
Yes that worked great! Thanks.